### PR TITLE
add possible value

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/9648

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -779,7 +779,7 @@ Accept wildcard characters: False
 ```
 
 ### -MeetingChatEnabledType
-Specify if users will be able to chat in meetings. Possible values are: Disabled, Enabled.
+Specify if users will be able to chat in meetings. Possible values are: Disabled, Enabled and EnabledExceptAnonymous.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:782]:
```suggestion
Specifies if users will be able to chat in meetings. Possible values are: Disabled, Enabled, and EnabledExceptAnonymous.
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 2]
```diff
@@ -553,7 +553,7 @@ Accept wildcard characters: False
 ```
 
 ### -MeetingChatEnabledType
-Specify if users will be able to chat in meetings. Possible values are: Disabled, Enabled.
+Specify if users will be able to chat in meetings. Possible values are: Disabled, Enabled and EnabledExceptAnonymous.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:556]:
```suggestion
Specifies if users will be able to chat in meetings. Possible values are: Disabled, Enabled, and EnabledExceptAnonymous.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@dariomws @tseward

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward Gentle reminder

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**